cerebrium/endpoints/custom-domains.mdx

```diff
@@ -0,0 +1,209 @@
+
+<Tabs>
+  <Tab title="POST Request">
+    ```bash
+    curl -X POST https://api.yourdomain.com/v4/p-1234/my-app/run \
+      -H "Authorization: Bearer {YOUR_API_KEY}" \
+      -H "Content-Type: application/json" \
+      -d "{"inputs": {"prompt": "Hello world"}}"
+    ```
+  </Tab>
+
+  <Tab title="GET Request">
+    ```bash
+    curl https://api.yourdomain.com/v4/p-1234/my-app/run \
+      -H "Authorization: Bearer {YOUR_API_KEY}"
+    ```
+  </Tab>
+</Tabs>
+
+**Multiple Apps in Same Project:**
+
+```bash
+# First app
+curl -X POST https://api.yourdomain.com/v4/p-1234/chatbot/predict
+
+# Second app
+curl -X POST https://api.yourdomain.com/v4/p-1234/image-generator/run
+
+# Third app
+curl -X POST https://api.yourdomain.com/v4/p-1234/data-processor/process
+```
+
+## Managing Your Domains
+
+### Viewing Domain Status
+
+In your project's Custom Domains tab, you can see:
+
+- **Domain Name**: The custom domain you've configured
+- **Status**: Current validation status (Pending, Validated, Failed)
+- **Actions**: Validate, view DNS records, or delete the domain
+
+### Domain Validation Statuses
+
+Cerebrium will attempt to validate your domain once every 30 minutes up to a maximum of 2 days. If your domain is not
+being validated, you can click on "Validation Status" to see the last validation attempt, along with any error that
+occurred.
+
+- **Pending**: Domain is waiting for DNS validation
+- **Validated**: Domain is successfully validated and ready to use
+- **Failed**: DNS validation failed - click "Validation Status" to see the specific error
+
+### Deleting Domains
+
+To remove a custom domain:
+
+1. Click the delete button (🗑️) next to the domain
+2. Confirm the deletion in the dialog
+3. The domain will be removed from Cerebrium's ingress controllers and SSL certificates cleaned up
+
+## Troubleshooting
+
+<Note>
+  For any questions or issues, reach out to us on our [Discord support
+  channel](https://discord.gg/ATj6USmeE2).
+</Note>
+
+### Validation Errors
+
+If your domain shows "Failed" status it means that the DNS record has been misconfigured, common issues include:
+
+- **Wrong target**: CNAME points to incorrect proxy URL
+- **Multiple conflicting records**: Remove any existing A records for the same hostname
+
+### Common DNS Provider Issues
+
+- **Cloudflare**: Disable proxy (set to "DNS only" - gray cloud icon)
+- **Route53**: Use simple routing policy, not weighted or latency-based
+- **Namecheap**: Use "@" for apex domains, not "www" or blank
+- **GoDaddy**: CNAME records cannot be used with apex domains - consider using a subdomain or an A record
+
+### Common DNS Mistakes
+
+- **Wrong proxy URL**: Ensure you're using the exact URL provided in the dashboard
+- **TTL too high**: Consider lowering TTL before making DNS changes
+- **Multiple records**: Remove conflicting A or CNAME records for the same name
+
+### Still Having Issues?
```

Comment:
I think you can remove this section. Enough instructions above and they can contact us

Reply:
Thanks, these are great